Makefile

```diff
@@ -3,7 +3,7 @@ NAMESPACE=pam
 NAME=oktapam
 BINARY=terraform-provider-${NAME}
 # On verion changes, update tag-checks.yml
-VERSION=0.1.5
+VERSION=0.2.1
```

Comment:
Whoops. Wonder if there's a way to add a check in Github to ensure this has been increased before making a tag.